### DISCUSSION TITLE
Release policy

### DISCUSSION THREAD

**Discussion by PhilippDehner** [gapitio/gapit-htmlgraphics-panel#177] [General] [answered]
Hi all, 

how often will releases be published? There are many new commits, but the last release is from 2022...

**Comment by ZuperZee** [comment, marked as answer]
The last commits are mostly for the website, but I try to make a release when the main code changes or if there are fixes that need to be released.

**Reply by PhilippDehner** [reply]
Thanks!

**Comment by PhilippDehner** [comment]
If I have a new feature, I can just create a fork and create a pull request?

**Reply by ZuperZee** [reply]
Yes

**Reply by ZuperZee** [reply]
What feature are you trying to make?
I suggest creating an issue first to make sure the feature is wanted.

**Reply by PhilippDehner** [reply]
I'm not sure if it should be a feature of this plugin or rather a feature of Grafana. In all cases where I use this plugin I need the data in rows, not in fields. So currently the "data" object contains the "fields" property, which includes the columns of a series. But I always need the rows. 

But thinking about this problem it is a Grafana problem...

**Reply by PhilippDehner** [reply]
https://github.com/grafana/grafana/issues/83815
